UnitTests/GitUI.Tests/ThreadHelperTests.cs

```diff
@@ -27,6 +27,7 @@ private static async Task ThrowExceptionAsync(Exception ex)
         }
 
         [Test]
+        [Ignore("Hangs")]
```

Comment:
💡 You can create an issue and assign it to me for this one

Reply:
👍 will do once this is merged